src/test/groovy/org/prebid/server/functional/tests/UserSyncSpec.groovy

```diff
@@ -60,4 +60,25 @@ class UserSyncSpec extends BaseSpec {
         IFRAME         || BLANK
         IFRAME         || PIXEL
     }
+
+    def "PBS should return uid in usersync url when uid is present but empty"() {
+        given: "Pbs config with usersync.#userSyncFormat.url"
+        var parameter = "?uid="
```

Comment:
1) `var`?
2) Looks like you're testing the wrong thing. Initial issue was that PBS was emitting a redirect url with `uid=null` because configuration didn't have `uid-macro` in the config.